src/main/scala/com/amazon/deequ/profiles/ColumnProfiler.scala

```diff
@@ -810,14 +810,15 @@ object ColumnProfiler {
       StringType, BooleanType, DoubleType, FloatType, IntegerType, LongType, ShortType
     )
     val originalStringNumericOrBooleanColumns = schema
-      .filter { field => validSparkDataTypesForHistograms.contains(field.dataType) }
+      .filter { field => validSparkDataTypesForHistograms.contains(field.dataType) ||
+        genericStatistics.typeOf(field.name) == Decimal }
```

Comment:
Why do we need this? We should be able to add `DecimalType` in the Set `validSparkDataTypesForHistograms`

Reply:
DecimalType can't just be added to validSparkDataTypesForHistograms, because it's a parameterized type (one could add something like DecimalType(10,10), but not an unparameterized instance). That's why I take the detour here.

The `genericStatistics.typeOf` is determined with a match-case, which actually does not care about the parameterization.
```
val knownType = field.dataType match {
          case ByteType | ShortType | IntegerType | LongType => Integral
          case FloatType | DoubleType => Fractional
          case DecimalType() => Decimal
```